server/controller/index.js

```diff
@@ -0,0 +1,6 @@
+const router = require('express').Router();
+const { validation } = require('../middleware/validation.js');
```

Comment:
@alaabadra It looks like validation.js is exporting a function called `isFound`, not `validation` - one of these files might need a change in their function name.